content/news/2020-11-13-release-plans-and-call-for-contrib.md

```diff
@@ -0,0 +1,48 @@
+type = "news"
+title = "R1 Status Update and Call for Contributors"
+author = "bitigchi"
+date = "2020-11-13 13:00:00+03:00"
+tags = ["r1"]
+
+After nearly 20 years of development, Haiku is very close to its first release. We need your help to make it happen. Here's a status update for R1, and latest project news.
+
+## Release Plans
+
+We are planning to release Beta3 sometime next year, hopefully followed by the first stable release, R1. We currently have [31](https://dev.haiku-os.org/query?status=assigned&status=in-progress&status=reopened&status=new&group=status&milestone=R1%2Fbeta3) tickets left for Beta 3, and [688](https://dev.haiku-os.org/query?status=assigned&status=in-progress&status=reopened&status=new&group=status&milestone=R1) tickets for R1.
+
+Haiku project is by no means a small project, it is a full operating system nonetheless. Maintaining a full-stack operating system is a huge work, and needs more than a handful of developers. Recently, one of our core developers, waddlesplash has recently announced his leave from the project due to personal reasons. waddlesplash has contributed to Haiku Project close to a decade, and has been the top contributor for 2017, 2018, and 2019. He has contributed invaluable additions to the Haiku Project, like the NVME driver, and USB3 support, with countless bug fixes and improvements on numerous Haiku components.
+
+**With this departure and the prolonged absence of the other core developers, currently we do not have the enough developer power to release R1. More beta releases are always possible, but without a stable release, Haiku will never have the chance to become a viable alternative for desktop users.**
+
+Making Haiku something more than an enthusiast operating system is not possible without your help.
+
+## Call for Developers
+
+If you are a C++ developer, we want you to help us in developing and maintaining Haiku. Our priority is to release R1, therefore we need urgent support in fixing R1 milestone bugs. Other than that, improvements on [these areas](https://www.haiku-os.org/community/getting-involved/developing) are necessary to provide a stable and usable user experience for Haiku users.
+
+## Why should I contribute to Haiku?
+
+- Haiku is open source, and licensed with a permissive license (MIT).
+- Haiku is inspired by BeOS, which is highly praised as an operating system; now, and back in the day.
+- Haiku offers a POSIX compatible operating system designed from scratch, with no legacy burden that other operating systems suffer from.
```

Comment:
POSIX in itself is a legacy burden. And BeOS is a second one. That's quite heavy legacy burden.

Reply:
Can you suggest a better wording, to make it sound more, like easier to port software, because of the POSIX compatibility?

Reply:
Remove the "with no legacy burden that other operating systems suffer from" because this project is about reimplementing a 20 year old OS using a compiler from the same era, and if that's not legacy burden, I don't know what is.

Reply:
Got it, I'll edit that part. What do you think of adding "clean codebase" instead? Is it accurate enough?

Reply:
Yes, focus on clean, readable and reusable code, and use of a reasonably high level language (C++, even if only C++98, is a large improvement over C for writing an OS, even if Linux Thorvalds wouldn't agree)